registry/handlers/layer.go

```diff
@@ -49,8 +49,8 @@ type layerHandler struct {
 // response.
 func (lh *layerHandler) GetLayer(w http.ResponseWriter, r *http.Request) {
 	ctxu.GetLogger(lh).Debug("GetImageLayer")
-	layers := lh.Repository.Layers()
-	layer, err := layers.Fetch(lh.Digest)
+	layerStore := lh.Repository.Layers()
+	layerReader, err := layerStore.Fetch(lh.Digest)
```

Comment:
Please don't make vanity changes to variable names. `layer` and `layers` are used consistently throughout the code to refer to a singular layer and collection of layers, respectively. The type information is inferred from the method call and should never be a part of the variable name. Completion should fill in the rest.

We are operating on the principles here: http://research.swtch.com/names and this is not up for discussion. This method is only 15 lines long and calling some thing `layerStore` adds no value over `layers`.